src/map/navi.cpp

```diff
@@ -139,7 +112,8 @@ static int add_path(struct node_heap *heap, int16 x, int16 y, int g_cost, struct
 	tp[i].f_cost = g_cost + h_cost;
 	tp[i].flag = SET_OPEN;
 	tpused[i] = 1 + (x << 16 | y);
-	heap_push_node(heap, &tp[i]);
+	heap.push_back(&tp[i]);
+	std::push_heap(heap.begin(), heap.end(), max_heap_comp);
```

Comment:
minheapcomp

Reply:
idk why, but not work if i put min heap comp at addpath